Kconfig

```diff
@@ -40,6 +40,14 @@ config DEBUG_PRINT_ON_UART1
       deck documentation on https://www.bitcraze.io/ for more information about
       pin allocation.
 
+config LIGHTHOUSE_AS_GROUNDTRUTH
+    bool "Use Lighthouse system as groundtruth"
+    default n
+    help
+      Use the lighthouse system as a groundtruth. This will also switch the
+      estimation method to the crossing beam position estimate and you can 
+      read out the positioning from the loggin lighthouse.x/.y/.z
+
```

Comment:
Great!

I think however we could change this to the Expansion deck menu in `src/deck/drivers/src/Kconfig` and add a dependency on the lighthouse deck:

```
config DECK_LIGHTHOUSE_AS_GROUNDTRUTH
    bool "Use Lighthouse system as groundtruth"
    depends on DECK_LIGHTHOUSE
    default n
    help
      Use the lighthouse system as a groundtruth. This will also switch the
      estimation method to the crossing beam position estimate and you can 
      read out the positioning from the loggin lighthouse.x/.y/.z
```

Reply:
Something like this?

Reply:
Or keep it in the menu but add the dependency.

Reply:
ah yeah but the define is not in the deck driver folder... wouldn't that be a problem?

Reply:
No all defines will be added to the generated `autoconf.h` no hierarchy.

Reply:
got it! I'll move it there. Expect new commits in a few minutes